cmake/test.cmake

```diff
@@ -288,21 +288,55 @@ finally:
     else()
         list(APPEND ALL_PATH_VARS
             LD_LIBRARY_PATH
+            IDIFF_LD_LIBRARY_PATH
+            LD_PRELOAD
         )
     endif()
 
     # Set initial empty values for all path vars
+    # NOTE - we prefix varnames with "MAYAUSD_VARNAME_" just to make collision
```

Comment:
Moved the comment to make it more relevant.

Reply:
I wanted to minimize changes in this pull request.  In a followup pull request for other changes, I will factor out much of this code and rename the prefix.